app/services/passporting/means_passporter.rb

```diff
@@ -34,6 +41,12 @@ def passport_types_collection
 
     private
 
+    def app_not_means_tested?
+      return true if crime_application.is_means_tested == 'no'
```

Comment:
To me `is_means_tested` sounds like it should be a predicate method, its implying a boolean outcome so should probably end in a `?` in line with the ruby convention. It would mean in situations like this we don't have to the comparision to 'no' as it would return `true`/`false`

Reply:
That's just the name I gave the value on the form, it made logical choice given the question wording, but I appreciate it breaks convention (and brain) so could change it. What do you think? - I was trying to follow the 'has_partner' pattern

https://staging.apply-for-criminal-legal-aid.service.justice.gov.uk/applications/68ce4681-3712-4f0e-add7-571a4557a919/steps/client/is_application_means_tested